openspec/explorations/zshrc-definitive-additions.md

```
@@ -0,0 +1,247 @@
+    fzf --preview 'git log --oneline --graph --date=short --pretty="format:%C(auto)%cd %h%d %s" {1} | head -20' |
+    sed 's/.* //' | sed 's#remotes/[^/]*/##')
+  if [[ -n "$branch" ]]; then
+    git checkout "$branch"
+  fi
+}
+```
+
+| Function | What | Why it's genuinely useful |
+|----------|------|--------------------------|
+| `fglog` | Interactive commit browser with full diff preview | Faster than `git log` for finding "that commit from last week" |
+| `fgco` | Branch picker with commit history preview | 50+ branches? This saves minutes vs. `git branch | grep` |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)